zyngui/zynthian_gui_fileselector.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#******************************************************************************
+# ZYNTHIAN PROJECT: Zynthian GUI
+# 
+# Zynthian GUI Step-Sequencer Class

```

Comment:
Mmmmm ... i think we could choose a better description for this piece of code ;-)